zabbix-ldap.conf.example

```diff
@@ -25,6 +25,7 @@ filtergroup = (&(objectClass=posixGroup)(cn=%s))
 filteruser = (&(objectClass=posixAccount)(uid=%s))
 groupattribute = memberUid
 userattribute = uid
+truncatedn = false
```

Comment:
Can you add this to the Readme instead?
(Including why this is needed and a n example of the truncation)?